### DISCUSSION TITLE
Help with 5G and LimeSDR

### DISCUSSION THREAD

**Discussion by andreas-geo** [srsran/srsRAN_Project#699] [General Help] [answered]
Hello everyone, I have successfully simulated 5G using Open5Gs and 4G UE and gNodeB from srsRAN project. I would like to try to simulate gNode B with LimeSDR USB and UE with a second LimeSDR. Has anyone else done this or maybe can offer some help ? 
Thanks in advance.

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @andreas-geo, 

srsRAN Project does not have support for LimeSDR. Only USRPs and O-RUs are supported as RF-frontends. That being said, as Lime uses Soapy, it might be possible to use the UHD/Soapy Wrapper. Other users have been able to get this approach to work with BladeRF devices, maybe you can do the same with Lime. Take a look at [this discussion post](https://github.com/srsran/srsRAN_Project/discussions/222#discussioncomment-7120314). 

As this is outside of the feature set of srsRAN Project I cannot guarantee it will work, but it may offer a potential work-around.
